blueprints/qunit/ember-cli-yadda/files/tests/unit/steps/steps.js

```diff
@@ -0,0 +1,14 @@
+import yadda from '../../helpers/yadda';
+
+export default function(assert) {
+  return yadda.localisation.English.library()
+    .given('I type "Ember g feature make-feature"', (next) => {
+      visit('/');
```

Comment:
`visit` is not available in the unittest world so it generates a failing test, which none of the other blueprints in ember do so I think we shouldn't either.